test/services/news/index.test.js

```diff
@@ -154,6 +154,40 @@ describe('news service', () => {
 				}
 			});
 
+			it('should respond with an error if a student is trying to access specific unpublished news', async () => {
+				const schoolId = (await createTestSchool())._id;
+				const now = Date.now();
+				const weekInMilSeconds = 604800000; // 1000 * 60 * 60 * 24 * 7
+				const user = await createTestUser({
+					schoolId,
+					roles: 'student',
+				});
+
+				const user2 = await createTestUser({
```

Comment:
use administratorUser and studentUser instead of numbering

Reply:
Done. Changed this for all test user variables in index.test.js for the news service.
Now it's only `studentUser`,`adminUser`,`teacherUser` or `noRoleUser`.